### ISSUE TITLE
HTML code <a href...> in error messages

### ISSUE THREAD

**Issue by thistlillo** [microsoft/vscode-jupyter#14330]
All of a sudden, Visual Studio Code started to report errors in Jupyter notebook with HTML links. For example, I see this: 

```
NameError                                 Traceback (most recent call last)
[/home/user/workspace/ws-code/202303/01_cluster.ipynb](https://vscode-remote+ssh-002dremote-002bhost-002eit.vscode-resource.vscode-cdn.net/home/user/workspace/ws-code/202303/01_cluster.ipynb) Cell 8 line 1
<a href='[vscode-notebook-cell://ssh-remote%2Bhost.it/home/user/workspace/ws-code/202303/01_cluster.ipynb#X10sdnNjb2RlLXJlbW90ZQ%3D%3D?line=0'>1</a>](vscode-notebook-cell://ssh-remote%2Bhost.it/home/user/workspace/ws-code/202303/01_cluster.ipynb#X10sdnNjb2RlLXJlbW90ZQ%3D%3D?line=0'%3E1%3C/a%3E) # scores

... plenty messages depending on the stack

NameError: name 'np' is not defined
```
So basically, you to not see the stack trace, but raw (not rendered) html code.

It started after the last update. Details:
Version: 1.82.1
Commit: 6509174151d557a81c9d0b5f8a5a1e9274db5585
Date: 2023-09-08T08:49:22.656Z
Electron: 25.8.0
ElectronBuildId: 23503258
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 22.6.0

on Mac OS.

Is there anything I can do to see the old beautifully rendered HTML code?

**Comment by FlorinAndrei** [comment]
Same here. Very annoying.

**Comment by adamw-u** [comment]
how and when to fix it!

**Comment by terry97-guel** [comment]
Same here... need fixing

**Comment by terry97-guel** [comment]
Well I think it's due to recent update.
Using previous version from https://code.visualstudio.com/updates/v1_81 fixed my problems.
After installation make sure to reinstall jupyter extension

**Comment by adamw-u** [comment]
Reload and update to the latest version can solve it.

**Comment by rebornix** [comment]
I think it's already fixed in latest VS Code by @amunger.

**Comment by amunger** [comment]
yes, should be fixed in 1.82.2
dupe of https://github.com/microsoft/vscode/issues/192605
